xen-posix/src/mini_libc.c

```diff
@@ -321,4 +323,10 @@ unsupported_function_crash(read);
 unsupported_function_crash(gmtime);
 unsupported_function_crash(strtod);
 unsupported_function_crash(rename);
-unsupported_function_crash(strerror);
+
+unsupported_function_log(size_t, fwrite, 1);
+unsupported_function_log(int, fputc, 1);
+unsupported_function_log(int, fputs, 1);
```

Comment:
With -Wall enabled, xen-ocaml/ocaml-src/byterun/io.c emits "implicit declaration" warnings for read, write, close, lseek, getpid, getppid.  These are declared in the unistd.h provided by xen-posix, but xen-ocaml will not `#include <unistd.h>` unless HAS_UNISTD is defined.

Adding -DHAS_UNISTD clears the declaration warnings, but the side-effect is that these additional functions need an implementation.  Should these stubs crash instead of log?